pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -352,7 +352,7 @@ func testNoChange(ctx context.Context, t *testing.T, testContext testrunner.Test
 func testUpdate(ctx context.Context, t *testing.T, testContext testrunner.TestContext, systemContext testrunner.SystemContext, resourceContext contexts.ResourceContext) {
 	// Tests with `SkipUpdate` explicitly set to 'true' or tests for
 	// auto-generated resources don't support update test.
-	if resourceContext.SkipUpdate || resourceContext.IsAutoGenerated(systemContext.SMLoader) {
+	if resourceContext.SkipUpdate || resourceContext.IsAutoGenerated(systemContext.SMLoader) && resourceContext.ResourceKind != "BigQueryConnectionConnection" {
```

Comment:
Just curious why we change the test behavior here. I think we should allow the dynamic test to cover the new update test(friendlyName &description) as well?

Reply:
This resource is hacky. It is not auto-generated, but we used the autogen channel to support it as a v1alpha1 resource. Auto-generated resources don't have the update test, so I have to explicitly make this resource a exceptional case to run testUpdate().

I don't want to turn this exception into a feature because this should be the only resource like this.

I'll add a comment about it.